crates/ruff_python_formatter/src/other/except_handler_except_handler.rs

```diff
@@ -2,12 +2,26 @@ use crate::comments::trailing_comments;
 use crate::expression::parentheses::Parenthesize;
 use crate::prelude::*;
 use crate::{FormatNodeRule, PyFormatter};
+use ruff_formatter::FormatRuleWithOptions;
 use ruff_formatter::{write, Buffer, FormatResult};
 use ruff_python_ast::node::AstNode;
 use rustpython_parser::ast::ExceptHandlerExceptHandler;
 
 #[derive(Default)]
-pub struct FormatExceptHandlerExceptHandler;
+pub struct FormatExceptHandlerExceptHandler {
+    has_star: bool,
```

Comment:
Nit: I think an enum `Starred` with the options `Yes` and `No` or `ExceptHandlerKind` with the variants `Starred` and `Regular` would improve readability on the `with_options` call site